cairo-vm-tracer/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "cairo-vm-tracer"
+version = "0.5.2"
+edition = "2021"
+license = "Apache-2.0"
+description = "Tracer to see step wise execution of Cairo instructions"
+
+[features]
+default = ["std"]
+std = []
+alloc = []
+tracer = []
+
+[dependencies]
+cairo-vm = { path = "../vm", version = "0.6.0" }
+thiserror = { workspace = true }
+num-bigint = { workspace = true }
+num-traits = { workspace = true }
+axum = "0.6.18"
+tokio = {version = "1.28.2", features = ["rt", "macros","rt-multi-thread"]}
+serde = { workspace = true }
+tower = { version = "0.4.13", features = ["util"] }
+tower-http = { version = "0.4.0", features = ["full"] }
+tracing = "0.1.37"
+tracing-subscriber = "0.3.17"
+include_dir = "0.7.3"
+mime_guess = "2.0.4"
+
+[dev-dependencies]
```

Comment:
1. We could simply remove the section if it's not used.
2. We require to always end our files with newlines, could you fix it?

Reply:
Fixed